<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Корзинка паролей</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icon-images.qrc">
    <normaloff>:/images/ProgramIcon</normaloff>:/images/ProgramIcon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionChangeCurrentPassword"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Правка</string>
    </property>
    <addaction name="actionEditAddFolder"/>
    <addaction name="actionEditAddPwd"/>
    <addaction name="separator"/>
    <addaction name="actionEditEdit"/>
    <addaction name="separator"/>
    <addaction name="actionEditDel"/>
    <addaction name="separator"/>
    <addaction name="actionCopyToClipboard"/>
    <addaction name="actionCopyLogin"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Вид</string>
    </property>
    <addaction name="actionViewExpand"/>
    <addaction name="actionViewUnExpand"/>
    <addaction name="separator"/>
    <addaction name="actionShowPwd"/>
    <addaction name="separator"/>
    <addaction name="actionViewPrimaryChecks"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>Работа с файлами</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionChangeCurrentPassword"/>
  </widget>
  <widget class="QToolBar" name="toolBarEdit">
   <property name="windowTitle">
    <string>Работа с содержимым</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEditAddFolder"/>
   <addaction name="actionEditAddPwd"/>
   <addaction name="actionEditEdit"/>
   <addaction name="actionEditDel"/>
   <addaction name="separator"/>
   <addaction name="actionCopyToClipboard"/>
  </widget>
  <widget class="QToolBar" name="toolBarView">
   <property name="windowTitle">
    <string>Отображение</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionViewExpand"/>
   <addaction name="actionViewUnExpand"/>
   <addaction name="separator"/>
   <addaction name="actionShowPwd"/>
   <addaction name="separator"/>
   <addaction name="actionViewPrimaryChecks"/>
   <addaction name="separator"/>
   <addaction name="actionHelpAbout"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/newfile.png</normaloff>:/images/images/newfile.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Создать...</string>
   </property>
   <property name="toolTip">
    <string>Создает новый файл без имени. При создании - запрашивает новый пароль для шифрования нового файла.</string>
   </property>
   <property name="statusTip">
    <string>Создание нового файла для хранения паролей</string>
   </property>
   <property name="whatsThis">
    <string>Создает новый файл без имени. При создании - запрашивает новый пароль для шифрования нового файла.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/openfile.png</normaloff>:/images/images/openfile.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Открыть...</string>
   </property>
   <property name="toolTip">
    <string>Открывает существующий файл с паролями. При загрузке нового файла, программа запросит пароль для доступа к нему.</string>
   </property>
   <property name="statusTip">
    <string>Открыть существующий файл с паролями</string>
   </property>
   <property name="whatsThis">
    <string>Открывает существующий файл с паролями. При загрузке файла, программа запросит пароль для доступа к нему.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохраняет текущий список паролей в файл, если файл уже определен, то сохраняет в него же, если файл еще неизвестен, то открывает окно выбора файла для сохранения.</string>
   </property>
   <property name="statusTip">
    <string>Сохранить текущий список паролей в файл</string>
   </property>
   <property name="whatsThis">
    <string>Сохраняет текущий список паролей в файл, если файл уже определен, то сохраняет в него же, если файл еще неизвестен, то открывает окно выбора файла для сохранения.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/saveas.png</normaloff>:/images/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить к&amp;ак...</string>
   </property>
   <property name="toolTip">
    <string>Сохраняет текущий список паролей в файл с новым именем.</string>
   </property>
   <property name="statusTip">
    <string>Сохранить текущий список паролей в новый файл</string>
   </property>
   <property name="whatsThis">
    <string>Сохраняет текущий список паролей в файл с новым именем.</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Вы&amp;ход</string>
   </property>
   <property name="toolTip">
    <string>Выход из программы</string>
   </property>
   <property name="statusTip">
    <string>Выход из программы</string>
   </property>
   <property name="whatsThis">
    <string>Выход из программы</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionEditAddFolder">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/addfolder.png</normaloff>:/images/images/addfolder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Добавить папку...</string>
   </property>
   <property name="toolTip">
    <string>Добавляет новую папку в дерево паролей.
Если в данный момент выделена уже существующая 
папка, то новая создается, как ветвь выделенной, 
иначе новая папка создается в корне дерева.</string>
   </property>
   <property name="statusTip">
    <string>Добавить новую папку в дерево.</string>
   </property>
   <property name="whatsThis">
    <string>Добавляет новую папку в дерево паролей.
Если в данный момент выделена уже существующая 
папка, то новая создается, как ветвь выделенной, 
иначе новая папка создается в корне дерева.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
  <action name="actionEditAddPwd">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/addrecord.png</normaloff>:/images/images/addrecord.png</iconset>
   </property>
   <property name="text">
    <string>Добавить паро&amp;ль...</string>
   </property>
   <property name="toolTip">
    <string>Добавляет новую запись в дерево паролей. 
Если в данный момент выделена уже существующая 
папка, то новая запись создается, как ветвь выделенной 
папки, иначе новая запись создается в корне дерева.</string>
   </property>
   <property name="statusTip">
    <string>Добавить новую запись</string>
   </property>
   <property name="whatsThis">
    <string>Добавляет новую запись в дерево паролей. 
Если в данный момент выделена уже существующая 
папка, то новая запись создается, как ветвь выделенной 
папки, иначе новая запись создается в корне дерева.</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actionEditEdit">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/editrecord.png</normaloff>:/images/images/editrecord.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Изменить пароль...</string>
   </property>
   <property name="toolTip">
    <string>Изменяет пароль для выделенной записи в дереве.</string>
   </property>
   <property name="statusTip">
    <string>Изменяет пароль для выделенной записи в дереве.</string>
   </property>
   <property name="whatsThis">
    <string>Изменяет пароль для выделенной записи в дереве.</string>
   </property>
   <property name="shortcut">
    <string>Shift+Return</string>
   </property>
  </action>
  <action name="actionEditDel">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/delete.png</normaloff>:/images/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удаляет выделенную папку или запись.
При удалении папки, также удаляется вся вложенная
в данную папку ветвь.</string>
   </property>
   <property name="statusTip">
    <string>Удалить выделенную папку или запись</string>
   </property>
   <property name="whatsThis">
    <string>Удаляет выделенную папку или запись.
При удалении папки, также удаляется вся вложенная
в данную папку ветвь.</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionShowPwd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/showpasswords.png</normaloff>:/images/images/showpasswords.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Показать пароли</string>
   </property>
   <property name="toolTip">
    <string>Режим переключения показать/скрыть пароли в дереве.
В случае включения режима показа программа попросит ввести текущий пароль пользователя, чтобы убедиться, что программой пользуется владелец файла.</string>
   </property>
   <property name="statusTip">
    <string>Режим показа/скрытия паролей в дереве</string>
   </property>
   <property name="whatsThis">
    <string>Режим переключения между видимость/скрытостью
паролей в дереве записей.
В случае включения режима показа программа попросит
ввести текущий пароль пользователя, чтобы убедиться,
что программой пользуется владелец файла.</string>
   </property>
  </action>
  <action name="actionChangeCurrentPassword">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/changepassword.png</normaloff>:/images/images/changepassword.png</iconset>
   </property>
   <property name="text">
    <string>С&amp;менить текущий пароль...</string>
   </property>
   <property name="toolTip">
    <string>Смена текущего (главного) пароля доступа к файлу.</string>
   </property>
   <property name="statusTip">
    <string>Смена текущего пароля доступа к файлу</string>
   </property>
   <property name="whatsThis">
    <string>Смена текущего (главного) пароля доступа к файлу.</string>
   </property>
  </action>
  <action name="actionCopyToClipboard">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/copy.png</normaloff>:/images/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Скопировать пароль в буфер обмена</string>
   </property>
   <property name="toolTip">
    <string>Копирует пароль текущей записи в буфер обмена.
Обычно данный режим используется, чтобы непосредственно
вставить пароль в другое приложение или веб-страницу.</string>
   </property>
   <property name="statusTip">
    <string>Скопировать пароль из выделенной записи в буфер обмена</string>
   </property>
   <property name="whatsThis">
    <string>Копирует пароль текущей записи в буфер обмена.
Обычно данный режим используется, чтобы непосредственно
вставить пароль в другое приложение или веб-страницу.</string>
   </property>
  </action>
  <action name="actionViewUnExpand">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/collapse.png</normaloff>:/images/images/collapse.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Свернуть все</string>
   </property>
   <property name="toolTip">
    <string>Cворачивает все дерево записей</string>
   </property>
   <property name="statusTip">
    <string>Свернуть все дерево записей</string>
   </property>
   <property name="whatsThis">
    <string>Cворачивает все дерево записей</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="actionViewExpand">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/expand.png</normaloff>:/images/images/expand.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Развернуть все</string>
   </property>
   <property name="iconText">
    <string>Развернуть все</string>
   </property>
   <property name="toolTip">
    <string>Разворачивает все дерево записей</string>
   </property>
   <property name="statusTip">
    <string>Развернуть все дерево записей</string>
   </property>
   <property name="whatsThis">
    <string>Разворачивает все дерево записей</string>
   </property>
   <property name="shortcut">
    <string>/</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/ProgramIcon</normaloff>:/images/ProgramIcon</iconset>
   </property>
   <property name="text">
    <string>Сведения &amp;о программе</string>
   </property>
   <property name="statusTip">
    <string>Открывает окно с информацией о разработчиках</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionHelpAboutQt">
   <property name="text">
    <string>Сведения о &amp;Qt</string>
   </property>
   <property name="statusTip">
    <string>Открывает окно с информацией об установленной версии Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Настройка</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionCopyLogin">
   <property name="text">
    <string>Скопировать имя в буфер обмена</string>
   </property>
  </action>
  <action name="actionViewPrimaryChecks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icon-images.qrc">
     <normaloff>:/images/images/stock_task.png</normaloff>:/images/images/stock_task.png</iconset>
   </property>
   <property name="text">
    <string>Режим выбора первичных записей</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icon-images.qrc"/>
 </resources>
 <connections/>
</ui>
